src/main/java/com/musseukpeople/woorimap/couple/application/InviteCodeService.java

```diff
@@ -28,4 +30,14 @@ public InviteCodeResponse createInviteCode(Long inviterId, LocalDateTime expireD
 
         return new InviteCodeResponse(inviteCode.getCode());
     }
+
+    public Long getInviterIdByInviteCode(String inviteCode) {
+        InviteCode foundInviteCode = inviteCodeRepository.findById(inviteCode)
+            .orElseThrow(() -> new NotFoundInviteCodeException(ErrorCode.NOT_FOUND_INVITE_CODE, inviteCode));
+        return foundInviteCode.getInviterId();
+    }
+
+    public void removeInviteCodeByCreatedCouple(Long inviterId, Long receiverId) {
+        inviteCodeRepository.deleteByTwoMemberId(inviterId, receiverId);
```

Comment:
```suggestion
        inviteCodeRepository.deleteByInIds(inviterId, receiverId);
```
twoMemberId는 뭔가 와닿지가 않네요. 너무 자세하다 해야하나??